client/src/components/InventoryTable.js

```diff
@@ -1,37 +1,45 @@
-import React from 'react'
+import React from 'react';
+
+var width = {
+  width: '100%',
+  textAlign: 'center',
+};
+
+const renderTable = items => {
+  console.log(items)
+  return items ?  (
+    <table style={width}>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Available</th>
+        <th scope="col">Name</th>
+        <th scope="col">Description</th>
+        <th scope="col">Owner</th>
+      </tr>
+      {items.map(item => (
+        <tr>
+          <th scope="row">{item.numberAvailable}</th>
+
+          <td><span className={item.numberAvailable > 0 ? ("oi oi-check text-success") : ("oi oi-x text-danger")}></span></td>
+          <td>{item.name}</td>
+          <td>{item.description}</td>
+          <td>{item.owner}</td>
+        </tr>
```

Comment:
Can you add an additional column that contains a simple '+' icon in it? This will be how users 'add' items to their cart. Make sure it updates the `quantity` number in the table. This is where you'll have to implement the 'Lifting State Up' principals I sent you earlier. If this is too difficult I can do it after everything else here is complete.